env/r3broot/dev.yaml

```diff
@@ -0,0 +1,27 @@
+spack:
+  definitions:
+    - when: platform == 'linux'
+      root: [root@6.16.00+fortran+gdml+http+memstat+pythia6+pythia8+vc~vdt+python+tmva ^mesa~llvm]
+    - when: platform == 'darwin'
+      root: [root@6.16.00+fortran+gdml+http+memstat+pythia6+pythia8+vc~vdt+python+tmva+aqua]
+  specs:
+    - pcre+jit
+    - python@2.7.16
+    - py-numpy@1.16.5
+    - googletest@1.8.1
+    - boost@1.68.0
+    - fairlogger
```

Comment:
Concretizer fails because version of FairLogger is not specified

Reply:
Interestingly enough, the concretizer only fails on mac. I have no idea, what's happening there. Anayway, added the version number from jun19.

Reply:
Was also failing in my Debian 8 container. Now is fine.